File: spring-boot-modules/spring-boot-basic-customization/src/main/java/com/baeldung/dispatchservlet/controller/Controller.java
package com.baeldung.dispatchservlet.controller;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping(value = "/")
public class Controller {

    @GetMapping
    public String getRequest(){
        return "Baeldung DispatcherServlet";
    }
}

File: spring-reactive-kotlin/src/main/kotlin/com/baeldung/springreactivekotlin/Controller.kt
package com.baeldung.springreactivekotlin

import org.springframework.http.MediaType
import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.ResponseBody
import reactor.core.publisher.Flux

@Controller
class Controller {

    @GetMapping(path = ["/numbers"], produces = [MediaType.APPLICATION_STREAM_JSON_VALUE])
    @ResponseBody
    fun getNumbers(): Flux<Int> {
        return Flux.range(1, 100)
    }

}
